packages/babel-plugin-transform-simplify-comparison-operators/__tests__/eqnull-test.js

```diff
@@ -0,0 +1,43 @@
+jest.autoMockOff();
+
+const babel = require("babel-core");
+const plugin = require("../src/index");
+
+function transform(code) {
+  return babel.transform(code, {
+    plugins: [plugin]
+  }).code;
+}
+
+describe("simplify-comparison-operators-plugin", () => {
+  it("should shorten `x == undefined`", () => {
+    const source = "x == undefined;";
+    const expected = "x == null;";
+    expect(transform(source)).toBe(expected);
+  });
+  it("should not shorten `x === undefined`", () => {
+    const source = "x === undefined;";
+    expect(transform(source)).toBe(source);
+  });
+
+  it("should shorten `undefined == null`", () => {
+    const source = "undefined == null";
```

Comment:
It [doesn’t appear to handle this](http://babeljs.io/repl/#?babili=true&evaluate=false&lineWrap=false&presets=react%2Cstage-0&targets=&browsers=&builtIns=false&debug=false&experimental=false&loose=false&spec=false&code=f(undefined%20%3D%3D%20null)&playground=true). Are you saying that I should move it there?